### DISCUSSION TITLE
Projection Integration Issues

### DISCUSSION THREAD

**Discussion by melhelbawy** [Eventuous/eventuous#117] [General]
Dear Alexy
I have an itegrated event using shovel eventuous 0.6.0, whe I tried to project the integrated event it is not update the mongoDB.

**Comment by alexeyzimarev** [comment]
I am not sure what your question means, sorry.

**Reply by melhelbawy** [reply]
Dear, Sorry for weak my description.

We have integrated event, that should affected in the subscribers Document project. 

**Ex**. If Customer update his Address, All started order should be updated for the new address.

The Q. Why that type projection not happend? the event stream integrated fine, but no projection updates !

**NOTE** 
 -- We use microservices with one MongoDb for all services.

**Reply by alexeyzimarev** [reply]
I am unsure how to answer this question as I don't know your code. UpdateMany works fine with MongoDB if you compose the MongoDB command correctly.
